eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/jdbc/JdbcEventStorageEngine.java

```diff
@@ -123,6 +174,19 @@ protected JdbcEventStorageEngine(Builder builder) {
      * <li>The {@code gapTimeout} defaults to an integer of size {@code 60000} (1 minute).</li>
      * <li>The {@code gapCleaningThreshold} defaults to an integer of size {@code 250}.</li>
      * <li>The {@code extendedGapCheckEnabled} defaults to {@code true}.</li>
+     * <li>The {@code createTokenAt} defaults to {@link Builder#createTokenAt}.</li>
```

Comment:
Nit: this piece of documentation should link to the util class I think, not the result of the builder.